README.md

```diff
@@ -149,6 +149,9 @@ Both installing the `newrelic-infra-operator` chart or spinning up the environme
 It is also possible to run such tests against any cluster you have access to by setting the environment variable
 `TEST_KUBECONFIG=/your/kube/config/path`. 
 
+E2E Testing (Metric NRQL queries):
```

Comment:
Can you please make this a heading like you did[ in this pr](https://github.com/newrelic/nri-kube-events/pull/342/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R102) so its' easier to find in the readme?

Reply:
I'll do that in PR #379 since I changed the wording of that paragraph